src/components/atoms/Plugin/IFrame/hooks.ts

```diff
@@ -137,8 +137,17 @@ export default function useHook({
     () => ({
       style: {
         display: visible ? undefined : "none",
-        width: visible ? (autoResize ? iFrameSize?.[0] : "100%") : "0px",
-        height: visible ? (autoResize ? iFrameSize?.[1] : "100%") : "0px",
+        // TODO: width iFrameSize?.[0]
+        width: visible
+          ? autoResize == "width-only" || autoResize == "both"
+            ? "100%"
+            : iFrameSize?.[0]
+          : "0px",
+        height: visible
+          ? autoResize == "height-only" || autoResize == "both"
```

Comment:
resizable = variable (not fixed at 100%)
not resizable = fixed at 100%

```suggestion
          ? autoResize == "height-only" || autoResize == "both"
            ? "100%"
            : iFrameSize?.[0]
          : "0px",
        height: visible
          ? autoResize == "width-only" || autoResize == "both"
```